### PR TITLE
#33 Params not useable

### PR DESCRIPTION
**Pull request by alexejFroebel**
### Definition of done:
- [x] The feature works -> production build
- [x] Tests exist and run successfully
- [x] All Tests work
- [x] Happy Path covered
- [x] No new Bugs/Followups
- [x] Implemented changes for the Groovy-API

- [x] Documentation:
  - Readme.md
  - Doc-Section in PIT
  - Groovy/OpenAPI

---

### Release Types:
- Minor
### Release notes:
- Added  2 params for getProductAttributeValues:
  - variantId
  - languageIds
---
## Code Review Checklist:
- [x] Does the code work?
- [x] Is all the code easily understood?
- [x] Are the function signatures named properly (function name, parameter names?)
- [x] Is the spelling correct?

- [x]  Is there redundant or duplicate code?
- [x]  Is there any commented out code?
- [x]  Can any logging or debugging code be removed?
- [x]  Is `for(i)` or `while` used? If so, remove
- [x]  Can any of the code be replaced with library functions?

-  Do you check input parameters (such as route parameters) for:
- [x] Type
- [x] Length
- [x] Format
- [x] Range?

- [x] Can comments be removed?
- [x] Is "external functionality" (API, Readme.md etc) documented/updated?
- [x] Is any unusual behavior or edge case handling described?

- [x] Do tests exist?
- [x] Do the snapshots look correct?
- [x] Are tests comprehensive?
- [x] Do unit tests actually test that the code is performing the intended functionality?

### CODE REVIEW

File: customizationArea/server/serviceMethods/pimApi.groovy [hunk 1]
```diff
@@ -439,6 +440,13 @@ class PitGroovyApi {
 
     /**
      * Retrieve all attributeValues of a Product
+     * @param  options Optional
+     * <p>
+     * Key: variantId - The id of the variant. If provided, the values are returned for the variant instead of the product
+     * <p>
+     * <p>
+     * Key: languageIds - All language-specific fields will be filtered to only include languages with the matching languageIds. If not provided, all language-specific fields are returned in all languages.
```
Comment (UweStolz) [inline customizationArea/server/serviceMethods/pimApi.groovy]: Delete the full stop
